components/ai_chat/resources/page/state/data-context-provider.tsx

```diff
@@ -165,15 +165,16 @@ function DataContextProvider (props: DataContextProviderProps) {
 
   const shouldShowLongPageWarning = React.useMemo(() => {
     if (
+      !isGenerating &&
       !hasDismissedLongPageWarning &&
```

Comment:
This new UI doesn't offer dismissal, right? So we can remove anything to do with that.

Reply:
removed